website/hugo.toml

```diff
@@ -41,3 +41,12 @@ pluralizelisttitles = false
     title = "Want to know more?"
     link_url = "https://redhat-appstudio.github.io/docs.appstudio.io/"
     link_text = "Konflux Documentation"
+
+[params.topbar]
+    enable = true
```

Comment:
```suggestion
[params.topbar]
    enable = true
    [params.topbar]
    enable = true
    text = "<img src='img/logo.svg'>"
```
This would add the logo to the topbar.
So we can remove the navbar

Reply:
Nice! I'll change accordingly.

Reply:
You can also update the `custom.css` to not display the navbar:
```
.navbar {
    display: none;
}
```

And afterwards maybe update the logo section in the toml file